src/wallet/index.ts

```diff
@@ -181,13 +216,89 @@ class Wallet {
   }
 
   /**
-   * Gets the classic address of the account this wallet represents.
+   * Gets the classic address of the account this wallet represents. This only is correct if this wallet corresponds
+   * to your [master keypair](https://xrpl.org/cryptographic-keys.html#master-key-pair). If this wallet represents a
+   * [regular keypair](https://xrpl.org/cryptographic-keys.html#regular-key-pair) this will provide an incorrect address.
```

Comment:
We need better support for regular keys in wallets anyways - the `getClassicAddress` is used in other places, such as in the Signer. There's a ticket for this.

Reply:
So should I remove the comment? (Or is that fine for now?)

Reply:
Keep the comment and add a TODO